doc/Writerside/topics/first_data_flow.md

```
@@ -0,0 +1,28 @@
+# Добавляем первый поток данных
+
+После установки и первоначальной настройки подсистемы, мы готовы к добавлению первого потока данных.
+
+## 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)